addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1177,6 +1358,129 @@ var SnippetEditor = Widget.extend({
             this.$el.css('pointer-events', '');
         }, 250);
     },
+    /**
+     * Called when the "send to back" overlay button is clicked.
+     *
+     * @private
+     * @param {Event} ev
+     */
+    _onSendBackClick(ev) {
+        ev.stopPropagation();
+        const rowEl = this.$target[0].parentNode;
+
+        // Finding the minimum z-index.
+        let minZindex = Number.MAX_SAFE_INTEGER;
+        for (const column of rowEl.children) {
+            minZindex = Math.min(minZindex, column.style.zIndex);
+        }
```

Comment:
Same suggestion I did earlier, this can be written as a one line and without the user of MAX_SAFE_INTEGER